libtcmu_log.c

```diff
@@ -411,10 +412,13 @@ static int output_to_fd(int pri, const char *timestamp,
 	if (pid <= 0)
 		return -1;
 
+	pthread_getname_np(pthread_self(), pname, 128);
```

Comment:
Same comment as in the other thread. The man page says this can only be 16 chars, so no need to do 128.

When we figure out what name we want to use for the device id, then we should maybe do a
#define TCMU_THREAD_NAME_LEN 16